src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
```

Comment:
minor 

you can move initialization to line 10 and just put the appropriate data in static block